### PR TITLE
Tag the examples generated by run_test! for easier filtering

### PR DESCRIPTION
**Pull request by asavageiv**
## Problem
rswag rspec examples aren't tagged, so in applications where there are a lot of request specs that aren't rswag specs a lot of time is wasted running those irrelevant specs.

## Solution
Tagging rspec examples allows for easy filtering in a custom rake task

### This concerns this parts of the Open API Specification:
N/A

### The changes I made are compatible with:
- [X] OAS2
- [X] OAS3
- [X] OAS3.1

### Related Issues
N/A

### Checklist
- [NA] Added tests
- [X] Changelog updated
- [X] Added documentation to README.md

### Steps to Test or Reproduce
cd test_app
rspec --tag rswag

### CODE REVIEW

File: README.md
Comment (lafferty151) [inline README.md]: > are tagged with the _`:rspec`_

`:rspec` --> `:rswag`

**Review by lafferty151 — COMMENTED** [review]
Why not just wrap your rswag specs in their own `describe` or `context` example group and add your `rswag` tag there?

Or put them into their own file(s) w/ a naming convention that allows you to filter them by `PATTERN=` or a custom .rspec or options file?

Do we really need this built into the gem itself?

**Comment by asavageiv** [comment]
The short answer is simplicity and scalability.

Wrapping the specs means requiring a developer to remember to do so. It's possible, but a slight burden. Renaming the files means breaking the existing file naming conventions.

The other route I considered was adding support to rswag for passing tags to `run_test!`.

**Comment by jtannas** [comment]
> Wrapping the specs means requiring a developer to remember to do so.

Agreed. I'm not a fan of solutions that depend on people consistently remembering to do something. Inevitably someone starts coding before having their morning coffee and things go sideways :upside_down_face:

**Review by jtannas — APPROVED** [review]
lgtm :+1: 
it's a minor code change that's very useful to those who want it
@BookOfGreg

**Comment by asavageiv** [comment]
Thanks jtannas. Just need approval from a maintainer now

**Pull request merged**
